src/LodeRunner.API/src/HealthChecks/CosmosHealthCheck.cs

```diff
@@ -60,7 +61,7 @@ public partial class CosmosHealthCheck : IHealthCheck
         /// <param name="logger">ILogger.</param>
         /// <param name="clientStatusService">the clientStatusService.</param>
         /// <param name="cosmosConfig">App CosmosConfig Interface.</param>
-        public CosmosHealthCheck(ILogger<CosmosHealthCheck> logger, IClientStatusService clientStatusService, ICosmosConfig cosmosConfig)
+        public CosmosHealthCheck(ILogger<CosmosHealthCheck> logger, ClientStatusService clientStatusService, ICosmosConfig cosmosConfig)
```

Comment:
Should this use the interface type instead of the concrete?

Reply:
Discussed it will be replacing concrete class with BaseService @jofultz

Reply:
@alfredoihernandez

Reply:
done